### PR TITLE
add sign tool

### PR DESCRIPTION
**Pull request by shrimalmadhur**
Fixes # .

### Motivation
Adding a sign tool to sign transactions for local testing

### CODE REVIEW

File: examples/configuration/sign.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "sign": {
+    "pub_key": {
+      "curve_type": "secp256k1",
+      "hex_bytes": "03c7e625aa08cad8f257d9ee2b9b7a0214f19f981afd5b498c728ad7ed6c0c3df6"
+    },
+    "private_key": "",
+    "signing_payload": {
+      "hex_bytes": "370e74254e8cbaa343af3564901456082ec7af967e45ff24ba061233b1a1b04f",
+      "signature_type": "ecdsa"
```
Comment (xiaying-peng) [inline examples/configuration/sign.json:10]: 2 types?

Comment (shrimalmadhur) [reply examples/configuration/sign.json:10]: `signature_type` and `curve_type`

File: cmd/sign.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+	"github.com/fatih/color"
+	"github.com/spf13/cobra"
+)
+
+var (
+	signCmd = &cobra.Command{
+		Use:   "sign",
+		Short: "Sign an unsigned payload with given private key",
+		Long: `Sign an unsigned payload with given private key
+				It supports Keypair specified by https://github.com/coinbase/rosetta-specifications`,
```
Comment (xiaying-peng) [inline cmd/sign.go:32]: Maybe u can refer to the example here ```examples/configuration/sign.json```

**Review by xiaying-peng — APPROVED** [review]

File: cmd/sign.go (hunk 2 above)
Comment (shrimalmadhur) [reply cmd/sign.go:32]: I will add a readme in next PR. :)

**Pull request merged**